ksp_plugin/flight_plan.cpp

```diff
@@ -124,8 +123,8 @@ bool FlightPlan::SetFinalTime(Instant const& final_time) {
 void FlightPlan::SetTolerances(
```

Comment:
Make this `SetAdaptiveParameters` (there's no reason to keep the integrator the same if we recompute everything).